java/HealthVault Mobile/HVSample/src/main/res/layout/profile.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+	xmlns:app="http://schemas.android.com/apk/res-auto"
+	android:layout_width="match_parent"
+	android:layout_height="match_parent">
+
+	<!-- Profile Image -->
+	<LinearLayout
```

Comment:
Use a FrameLayout if we are just containing a single element. LinearLayout has more advanced layout characteristics which are not needed in this case.